tools/hasher/src/main/java/org/apache/shiro/tools/hasher/Hasher.java

```diff
@@ -70,7 +70,7 @@ public final class Hasher {
     private static final Logger LOG = LoggerFactory.getLogger(Hasher.class);
 
     private static final String HEX_PREFIX = "0x";
-    private static final String DEFAULT_ALGORITHM_NAME = "MD5";
+    private static final String DEFAULT_ALGORITHM_NAME = "SHA-256";
```

Comment:
We need to document this on shiro-site.

Reply:
Done. See https://github.com/apache/shiro-site/pull/249